### PR TITLE
API-5504: Java 11 update and repo alignment (scalaFix & scalaFmt)

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -13,6 +13,17 @@ lazy val appName = "api-simulator"
 lazy val appDependencies: Seq[ModuleID] = compile ++ test
 lazy val bootstrapVersion = "7.12.0"
 
+ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
+
+inThisBuild(
+  List(
+    scalaVersion := "2.12.15",
```
Comment (johnsgp) [inline build.sbt]: I think this should match the version at line 61.

File: scalastyle-config.xml [hunk 2]
```diff
@@ -10,12 +10,6 @@
     <check level="warning" class="org.scalastyle.scalariform.SpacesAfterPlusChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.file.WhitespaceEndOfLineChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"/>
-    <check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
```
Comment (johnsgp) [inline scalastyle-config.xml:13]: I didn't think we were removing this.

File: build.sbt (hunk 1 above)
Comment (mattclark-zerogravit) [reply build.sbt]: yep, fixed cheers. I tried changing both to 2.12.15 and it broke the build so I just aligned them to 2.12.12.
Thanks for spotting!

File: scalastyle-config.xml (hunk 2 above)
Comment (mattclark-zerogravit) [reply scalastyle-config.xml:13]: I think you can either remove it or set it to false. I've added it back in but set it to false

**Review by syed-ali-hmrc — APPROVED** [review]

**Pull request merged**